server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1478,20 +1477,9 @@ public Response bind(
             entitlements = entitler.bindByPool(poolIdString, consumer, quantity);
         }
         else {
-            try {
-                AutobindData autobindData = AutobindData.create(consumer).on(entitleDate)
-                        .forProducts(productIds).withPools(fromPools);
-                entitlements = entitler.bindByProducts(autobindData);
-            }
-            catch (ForbiddenException fe) {
-                throw fe;
-            }
-            catch (CertVersionConflictException cvce) {
-                throw cvce;
-            }
-            catch (RuntimeException re) {
-                log.error("Autobind error", re);
-            }
+            AutobindData autobindData = AutobindData.create(consumer).on(entitleDate)
+                    .forProducts(productIds).withPools(fromPools);
+            entitlements = entitler.bindByProducts(autobindData);
```

Comment:
As we catch and throw without even logging, there is no functional reason for this. Is it a placeholder in the code?